lib/src/gateway/shard.dart

```diff
@@ -52,7 +52,7 @@ class Shard extends Stream<ShardMessage> implements StreamSink<GatewayMessage> {
       } else if (message is EventReceived) {
         final event = message.event;
 
-        if (event is! RawDispatchEvent) {
+        if (event is GatewayEvent) {
```

Comment:
Changing this line modifies the logic so it is now incorrect - `event` is always a `GatewayEvent`, so the `else` branch that outputs more information for dispatch events is always skipped, which reduces verbose logging output.